mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalBatchConsumerVerticle.java

```diff
@@ -238,11 +268,13 @@ private Flowable<Pair<Optional<Bundle>, Collection<BatchableJournalRecord>>> lis
       throw new IllegalStateException("KafkaConsumer not initialized");
     }
     return kafkaConsumer.toFlowable()
+      .onBackpressureBuffer(10_000,
+        () -> LOGGER.error("listenKafkaEvents:: Backpressure buffer full"),
+        BackpressureOverflowStrategy.DROP_OLDEST)
```

Comment:
just to clarify, if we discard old events, they won't be lost, right? as their offsets were not committed to broker, and broker will still send those discard events later on?

Reply:
They would be lost if new offsets are committed to the database.

I read the code for the FlowableReadStream in vert.x and it looks like the kafka consumer is automatically paused when there is backpressure. So please remove this change and lets rely on the buffer from vertx. https://github.com/vert-x3/vertx-rx/blob/96287695ac0a58ad01bf9331088ca69242d87564/rx-java3-gen/src/main/java/io/vertx/rxjava3/impl/FlowableReadStream.java#L72